rfcs/013-release_deployment_tracking/v1/README.md

```diff
@@ -0,0 +1,181 @@
+# Version 1 
+
+## Implementation from 24/01/2019
+
+This solution describes a set of steps to track build, release and deployment but is deliberately agnostic of deployment mechanism. 
+
+We make use of SSM to track which release should deploy into which environment. Out terraform stacks read from these SSM parameters to get which images to apply.
+
+Terms defined in this document apply only here.
+
+### Build artifacts
+
+> **Build artifact:** A container image encapsulating application code that can be configured by environment variable. The artifact _does not_ contain configuration.
+
+When container images intended for release are built they will be added to an ECR repository in AWS, by our build tooling.
+
+#### Build artifact SSM Parameters
+
+The URI of the container image will then be used to update an SSM parameter with a key:
+
+ `/{project_id}/images/{label}/{service_id}`
+
+The attributes are described as follows:
+- **project_id**: an identifier for a project.
+- **label:** A label for the set of images released, e.g. latest, stable, v2-branch
+
+ For example:
+
+ `/storage/images/latest/archivist`
+
+SSM parameters provide a versioned record of build artifacts. SSM allows descriptions to be added to updates, these descriptions should contain the `user_id` of what or who is updating the version
+
+This mechanism is provided via a python application, packaged in a docker container distributed via the https://github.com/wellcometrust/dockerfiles repository.
+
+### Project structure
+
+> **Project:** A set of services that when composed perform a function.
```

Comment:
This definition is incredibly generic. It would help to provide some examples, and maybe this is more specific:

```suggestion
> **Project:** A set of services that are deployed together.
```